spark/src/main/scala/geotrellis/spark/tiling/MapKeyTransform.scala

```diff
@@ -76,14 +78,20 @@ class MapKeyTransform(val extent: Extent, val layoutCols: Int, val layoutRows: I
     GridBounds(colMin, rowMin, colMax, rowMax)
   }
 
+  def boundsToExtent(gridBounds: GridBounds): Extent = apply(gridBounds)
+
   def apply(gridBounds: GridBounds): Extent = {
     val e1 = apply(gridBounds.colMin, gridBounds.rowMin)
     val e2 = apply(gridBounds.colMax, gridBounds.rowMax)
     e1.expandToInclude(e2)
   }
 
-  def apply(p: Point): SpatialKey =
-    apply(p.x, p.y)
+  def pointToKey(p: Point): SpatialKey = apply(p)
+
+  def apply(p: Point): SpatialKey = apply(p.x, p.y)
+
+  /** Fetch the [[SpatialKey]] that corresponds to some coordinates in some CRS on the Earth. */
+  def pointCoordsToKey(x: Double, y: Double): SpatialKey = apply(x, y)
```

Comment:
This feels like a place where overloading is helpful to overload between `pointToKey(Point(3,5))` and `pointToKey(x=3,y=5)` are both unsurprising and more discoverable.

Reply:
This I've added.